ibm_catalog.json

```diff
@@ -25,22 +25,23 @@
         "observability agents"
       ],
       "short_description": "Configures IBM Cloud Observability resources",
-      "long_description": "Solutions which support configuring IBM Cloud Observability resources for logging, monitoring and activity tracking.",
+      "long_description": "Solutions that support configuring IBM Cloud Observability resources for logging, monitoring, and activity tracking.",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-da/main/images/observability-icon.svg",
       "provider_name": "IBM",
       "features": [
         {
-          "title": "Configures Log Analysis",
-          "description": "Configures Log Analysis instance and agents."
+          "title": "Configures IBM Log Analysis",
+          "description": "Configures the IBM Log Analysis instance and logging agents."
         },
         {
-          "description": "Configures Cloud Monitoring",
-          "title": "Configures Cloud Monitoring. instance and agents."
+          "title": "Configures the IBM Cloud Monitoring instance and logging agents.",
+          "description": "Configures IBM Cloud Monitoring"
```

Comment:
I think title should be `Configures IBM Cloud Monitoring`
and description should be `Configures the IBM Cloud Monitoring instance and logging agents.` based on how it is written above for Log Analysis.

Reply:
@iamar7 for some reason, I can no longer edit the file directly in the PR. I've created [a PR](https://github.com/acolleen/terraform-ibm-observability-da/pull/1) to  Colleen's fork and branch to show up here. She is out until tomorrow, according to Slack.